lib/request-handler.js

```diff
@@ -0,0 +1,148 @@
+"use strict";
+
+
+/**
+ * @typedef {Object} requestHandler
```

Comment:
I think that in general, these factories produce something more like `requestHandler~result` instead of `requestHandler` things.  Or perhaps `{opentoken-lib-js~requestHandler}` to avoid potential namespace collisions.  What do you think?